pipeline/pipeline.py

```diff
@@ -20,9 +21,10 @@ class Pipeline(object):
     Pipeline methods return the pipeline object, allowing
     for methods to be chained together.
     '''
+
     def __init__(
-        self, name, display_name, settings_file=None,
-        log_status=True, conn=None
+            self, name, display_name, settings_file=None,
+            settings_from_file=True, log_status=False, conn=None,
```

Comment:
There might be a more elegant way of indicating not to use any settings_file and using a default location.